frontend/public/components/edit-yaml.jsx

```diff
@@ -585,21 +590,70 @@ export const EditYAML = connect(stateToProps)(
       });
     }
 
+    toggleSidebar = () => {
+      this.setState((state) => {
+        return { showSidebar: !state.showSidebar };
+      });
+      window.dispatchEvent(new Event('sidebar_toggle'));
+    };
+
     render() {
       if (!this.props.create && !this.props.obj) {
         return <Loading />;
       }
 
-      const { connectDropTarget, isOver, canDrop } = this.props;
+      const { connectDropTarget, isOver, canDrop, create, yamlSamplesList } = this.props;
       const klass = classNames('co-file-dropzone-container', {
         'co-file-dropzone--drop-over': isOver,
       });
 
-      const { error, success, stale, yaml, height } = this.state;
-      const { create, obj, download = true, header } = this.props;
+      const { error, success, stale, yaml, height, showSidebar } = this.state;
+      const { obj, download = true, header } = this.props;
       const readOnly = this.props.readOnly || this.state.notAllowed;
       const options = { readOnly, scrollBeyondLastLine: false };
       const model = this.getModel(obj);
+      const samples = getResourceSidebarSamples(model, yamlSamplesList);
+      const showSamples = !_.isEmpty(samples) && create;
+      const linkLabel = `View Schema ${showSamples ? 'and Samples' : ''}`;
+      const sidebarLink = (
+        <div className="co-p-has-sidebar__sidebar--hidden hidden-sm hidden-xs">
```

Comment:
Rather than hiding each individual link, hide the parent node (`yaml-editor__icons`) so you don't have to duplicate.

`co-p-has-sidebar__sidebar--hidden` isn't a great className.  Recommend a shared class to use with the other link.